spec/controllers/videos_controller_spec.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe VideosController do
+  describe "GET show" do
+
+    context "with authenticated users" do 
+      before do
+        #user = User.create(email: 'user@example.com', password: 'password', full_name: "Example" )
+        session[:user_id] = Fabricate(:user).id #user.id
+      end
+
```

Comment:
If you are going to use a before block you really ought to have at least two tests. Which you can actually do here, you just have to bring the other authenticated users test into the right context block. Also the comments on 8, 9, 13 should get deleted.